jepsen/src/jepsen/checker.clj

```diff
@@ -359,20 +359,18 @@
 (defn counter
   "A counter starts at zero; add operations should increment it by that much,
   and reads should return the present value. This checker validates that at
-  each read, the value is at greater than the sum of all :ok increments, and
-  lower than the sum of all attempted increments.
+  each read, the value is at greater than the sum of all :ok increments and
+  :invoke decrements, and lower than the sum of all attempted increments and
```

Comment:
use :invoke/attempted for both increments/decrements. Unless they are meant to mean different things.